Describe the TagBox demo overview (#749)
DevExpress/devextreme-demos: JSDemos/Demos/TagBox/Overview/description.md

The [TagBox](/Documentation/ApiReference/UI_Components/dxTagBox/) allows users to select multiple items from a drop-down list. This demo illustrates the following TagBox properties:

- [items](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/items/)  
  An array of items displayed by the TagBox.

- [searchEnabled](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#searchEnabled)  

>   Specifies whether the search is enabled

Reviewer: ```suggestion
  Specifies whether the search is enabled.
```

- [showSelectionControls](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#showSelectionControls)  

>   Defines whether selection controls are displayed

Reviewer: I suggest you to remove the passive voice. I also think that we can be more precise here and say 'selection checkboxes' to clarify what kind of controls we use.

```suggestion
  Defines whether to display the selection controls.
```

- [applyValueMode](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#applyValueMode)  
  Specifies whether selected values are applied instantly or when a user clicks the **OK** button.

- [hideSelectedItems](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#hideSelectedItems)  

>   Specifies whether items are removed from the drop-down list upon selection.

Reviewer: ```suggestion
  Specifies whether the selected items are removed from the drop-down list.
```

- [multiline](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#multiline)  
  Displays selected items on single or multiple lines.

- [acceptCustomValue](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#acceptCustomValue)  

>   Specifies whether users can enter custom values. Requires the [onCustomItemCreating](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#onCustomItemCreating) implementation.

Reviewer: We should not use articles before the property names alone. I see that you also use articles with the component names, which is strange to me, but I also see that Roman and our correctors are OK with it. Anyway, let's change the sentence as follows:

```suggestion
  Specifies whether users can enter custom values. Requires that you also implement the [onCustomItemCreating](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#onCustomItemCreating) handler.
```

- [placeholder](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#placeholder)  
  Customizes the TagBox placeholder.

- [value](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#value)  

>   Specifies items the TagBox displays.

Reviewer: This description says the same thing as the `items` description. It seems like you should change one of them or both.

Author: Why reinvent the wheel? The description here is straight to the point while the docs expand on it further

Reviewer: I totally agree. But this description is almost the same as 'An array of items displayed by the TagBox.' from the first point. It is hard to understand the difference. I think the following change will make this property's purpose more clear:

```suggestion
  Specifies items the TagBox currently displays.
```

- [disabled](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#disabled)  

>   Defines whether the component resopnds to user interaction.

Reviewer: ```suggestion
  Defines whether the component responds to user interaction.
```

- [dataSource](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#dataSource)  

>   Binds the TagBox to data. Unlike the **items** property, **dataSource** accepts the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object that allows users to sort, filter, group, and shape data.

Reviewer: I would say that wee need to show more differences from the `items` property in terms of data binding. Maybe it is also a good idea to say that we cannot use these properties simultaneously.

> - [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)  

Reviewer: ```suggestion

- [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)  
```

>   Allows you to customize the text field and drop-down list items.

Reviewer: The [fieldTemplate](https://js.devexpress.com/Documentation/21_1/ApiReference/UI_Components/dxTagBox/Configuration/#fieldTemplate) property allows users to customize the text field.

Author: itemTemplate is used in this demo https://js.devexpress.com/Demos/WidgetsGallery/Demo/TagBox/Overview/jQuery/Light/

Reviewer: Right you are. But does it customize the text field?

> - [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)  

Reviewer 2: ```suggestion

- [itemTemplate](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#itemTemplate)  
```
